test/functional/segwit.py

```diff
@@ -70,7 +70,7 @@ def getutxo(txid):
 
 def find_unspent(node, min_value):
     for utxo in node.listunspent():
```

Comment:
nit: Can use `query_options={"minimumAmount": min_value}`

Reply:
Not sure I understand the suggestion.

Reply:
`listunspent` takes named arguments according to the documentation, so you could add the named argument and remove some code from the python script instead.